### DISCUSSION TITLE
Voice control? 🤔

### DISCUSSION THREAD

**Discussion by ecseditamas** [eritislami/evobot#708] [Ideas]
According to me, voice control would be a cool feature.
I found a DiscordSpeechBot project which works perfectly.
https://github.com/healzer/DiscordSpeechBot

**Comment by wolf-yuan-6115** [comment]
> According to me, voice control would be a cool feature.
> I found a DiscordSpeechBot project which works perfectly.
> https://github.com/healzer/DiscordSpeechBot

We are not god
